### PR TITLE
Check elasticsearch data

### PR DESCRIPTION
**Pull request by zhquan**
check datafreshness, identities, uuid, author_name, and email address

### CODE REVIEW

File: es_author_check_grt_date.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            #indexes = Config[project]['indexes'].replace(", ", ",")
+            #conf['backends'] = indexes.split(',')
+            conf['es'] = Config[project]['es']
+            conf['user'] = Config[project]['user']
+            conf['password'] = Config[project]['password']
+
+    return conf
+
+def main():
+    tiempo_1 = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: Please use English for variables

File: es_author_check_grt_date.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    args = read_arguments()
+
+    conf_name = args.config_file
+
+
+    conf = get_conf(conf_name)
+
+    indexes = get_indexes(conf)
+
+    tiempo_inicio_scan = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: Please use English for your variables

File: es_author_check_grt_date.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    authors_all = check_authors(conf, indexes)
+    tiempo_final_scan = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: Please use English for your variables

File: es_author_check_grt_date.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    return df
+
+def get_conf(conf_name):
+    Config = configparser.ConfigParser()
+    Config.read(conf_name)
+
+    conf = {}
+    for project in Config.sections():
+        if project != "notification":
+            #indexes = Config[project]['indexes'].replace(", ", ",")
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: Please remove this if this is not needed in the code

File: es_author_check_grt_date.py [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    return args
+
+def get_indexes(conf):
+    # Get the indexes into a list(indexes)
+    database = "curl -k -u "+conf['user']+":"+conf['password']+" 'https://"+conf['es']+"/_cat/aliases' 2>/dev/null | awk 'BEGIN {FIELDWIDTHS = "+'"2 1"'+"} {print $2}' | uniq"
+    indexes = (os.popen(database).read().split('\n'))
+
+    return indexes
+
+def check_authors(conf, backends):
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: Please add comments with the purpose of this function.

You're checking authors, but what exactly? looking for empty ones? @? oddities in the names?

File: es_author_check_grt_date.py [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+    vc_id = no_duplicates.a_id.value_counts(dropna=False)
+    id_duplicated = json.loads(vc_id[vc_id > 1].to_json())
+    for id in id_duplicated:
+        name_list =  no_duplicates[no_duplicates.a_id==id].a_name.unique().tolist()
+        id_duplicated[id] = name_list
+    if "null" in id_duplicated:
+        if len(id_duplicated["null"]) == 0:
+            del id_duplicated["null"]
+    write_file("id_duplicated.json", id_duplicated)
+    tiempo_final = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: English for variables

File: es_author_check_grt_date.py [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+    message += "\nCreate id_duplicated.json: "+str(tiempo_final-tiempo_inicio)
+
+    tiempo_inicio = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: English for variables

File: es_author_check_grt_date.py [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+    authors_a = authors[authors.a_name.str.contains("@")].to_json()
+    write_file("authors_a.json", json.loads(authors_a))
+    tiempo_final = time()
```
Comment (dicortazar) [inline es_author_check_grt_date.py]: English for variables

File: es_datafreshness_jenkins.py [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Quan Zhou <quan@bitergia.com>
+#
+
+
```
Comment (dicortazar) [inline es_datafreshness_jenkins.py]: Please add in some place the purpose of this script.

File: identities_check.py [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Quan Zhou <quan@bitergia.com>
+#
+
```
Comment (dicortazar) [inline identities_check.py]: Please add the purpose of this script in some place

File: identities_check.py [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+import json
+import sys
+
+def openFile(fileName):
```
Comment (dicortazar) [inline identities_check.py]: Please use similar ways to name a function across the scripts. As an example you're using here openFile, but then in others you're using get_xxx_yyy. 

You can follow as an example PEP8 at https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

**Comment by zhquan** [comment]
I close this PR and I will create PR for each script

**Pull request closed without merging**